octoprint_mrbeam/templates/settings/debug_settings.jinja2

```diff
@@ -7,8 +7,8 @@
             <div class="controls">
                 <label class="checkbox">
                     <input type="checkbox" data-bind="checked: settings.plugins.mrbeam.terminal">
-                    <strong>Enable terminal</strong>
-                    <div>Requires full page reload: <strong>Shift + F5</strong> (Windows and Linux) or <strong>Ctrl + Shift + r</strong> (Mac)</div>
+                    <strong>{{ _('Enable terminal') }}</strong>
+                    <div>{{ _('Requires full page reload:') }} <strong>Shift + F5</strong> {{ _('(Windows and Linux) or') }} <strong>Ctrl + Shift + r</strong> (Mac)</div>
```

Comment:
can be one string with the strong-tags included ;-)

Reply:
ok